webpack/assets/javascripts/react_app/components/Layout/components/UserDropdowns.js

```diff
@@ -13,20 +14,21 @@ const UserDropdowns = ({
   changeActiveMenu,
   notificationUrl,
   ...props
-}) => (
-  <VerticalNav.IconBar {...props}>
-    <NavItem
-      className="drawer-pf-trigger dropdown notification-dropdown"
-      id="notifications_container"
-    >
-      <NotificationContainer data={{ url: notificationUrl }} />
-    </NavItem>
-    {!!user &&
-      !!user.current_user && (
+}) => {
+  const userInfo = get(user, 'current_user.user');
```

Comment:
I wonder if it wont make our life easier that the parent component will do all of the extraction and simply pass a user object as expected?